### PR TITLE
Minor grammatical corrections

### PR DESCRIPTION
**Pull request by andrew-dixon**
A couple of minor grammatical corrections I noticed while reading the page.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/2016-10-31/schedule/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+`AWS::Lambda::Function` is the top-level CloudFormation resource to define an Amazon Lambda function. Because we want to schedule the function’s periodic execution, we include an `Events` property on our `AWS::Serverless::Function` resource. This allows us to define the function execution schedule *within* the context of the function’s properties. Behind-the-scenes, the `Events` property expands into a `AWS::Events::Rule` resource with an invocation rate of once every 5 minutes.
+
+Lastly, in order for the CloudWatch Events API to invoke our function, it needs permissions to do so. `AWS::Lambda::Permission` grants CloudWatch Events the permission to invoke our function.
+
+### Deployment
+
+The [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli) builds on top of the SAM specification by providing a single tool to manage the packaging and deployment of serverless applications. Once the `sam` tool is installed, the application deployment process occurs in two phases.
+
+First, use `sam` to upload the binary to S3 and reference it in a newly created `packaged.yaml` CloudFormation configuration.
```
Comment (SoodDhruv) [inline examples/2016-10-31/schedule/README.md]: First, use `sam package`

File: examples/2016-10-31/schedule/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              --output-template-file packaged.yaml
+```
+
+Before using `sam` to deploy using the contents of `packaged.yaml`, run a quick `diff` to see what changed.
+
+```bash
+$ diff template.yaml packaged.yaml
+```
+
+Lastly, use `sam` again to deploy the template through a CloudFormation stack named `Test`.
```
Comment (SoodDhruv) [inline examples/2016-10-31/schedule/README.md]: Change "Lastly, use `sam` again" to "Second, use `sam package`

File: examples/2016-10-31/schedule/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- `AWS::Lambda::Permission`
+- `AWS::Events::Rule`
+
+`AWS::Lambda::Function` is the top-level CloudFormation resource to define an Amazon Lambda function. Because we want to schedule the function’s periodic execution, we include an `Events` property on our `AWS::Serverless::Function` resource. This allows us to define the function execution schedule *within* the context of the function’s properties. Behind-the-scenes, the `Events` property expands into a `AWS::Events::Rule` resource with an invocation rate of once every 5 minutes.
+
+Lastly, in order for the CloudWatch Events API to invoke our function, it needs permissions to do so. `AWS::Lambda::Permission` grants CloudWatch Events the permission to invoke our function.
+
+### Deployment
+
+The [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli) builds on top of the SAM specification by providing a single tool to manage the packaging and deployment of serverless applications. Once the `sam` tool is installed, the application deployment process occurs in two phases.
```
Comment (SoodDhruv) [inline examples/2016-10-31/schedule/README.md]: Please change this to:
The [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli) lets you locally build, test, and debug serverless applications defined by AWS SAM templates. You can also use ‘sam cli’ to deploy your application. [Install sam cli](https://docs.aws.amazon.com/lambda/latest/dg/sam-cli-requirements.html), and then deploy your application in two steps:

**Comment by brettstack** [comment]
@andrew-dixon I merged develop in as there was a conflict from another PR I just merged into develop. Could you take a look at the comments from @SoodDhruv and we'll get this merged in. Thanks!

**Comment by andrew-dixon** [comment]
@brettstack Not sure if I'm missing something but I don't see any comments.

**Comment by SoodDhruv** [comment]
@andrew-dixon yeah, those comments were for examples/2016-10-31/schedule/README.md. I've submitted a PR to address those. 

Your changes to HOWTO.md look good. Thank you for your contribution!

**Pull request merged**